src/styles/blocks/card.scss

```
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &:not(:nth-child(4n)) {
+    margin-right: $card-space;
+  }
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
```

```suggestion
  border-radius: 5px;
  
  &:not(:nth-child(4n)) {
    margin-right: $card-space;
  }
  
  &:not(:nth-last-child(-n + 4)) {
    margin-bottom: 46px;
  }
```

add indentions here as well to make the code more readable